Source/SpeechToTextV1/Models/TrainingWarning.swift

```diff
@@ -24,29 +24,29 @@ public struct TrainingWarning: Codable, Equatable {
     /**
      An identifier for the type of invalid resources listed in the `description` field.
      */
-    public enum WarningID: String {
+    public enum Code: String {
```

Comment:
This is troubling.  This looks like a breaking change.

Reply:
@mediumTaj see @mkistler 's comments above about the `TrainingWarning` model. It looks like there are some breaking changes that came out of the generator for this. Does this require hand edits, or is this a known change to the API definition?

Reply:
@mkistler it looks like these changes can be accepted as this model does not get used at runtime.